README.md

```diff
@@ -1,70 +1,52 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Calculator App
+A simple calculator application developed in ReactJS which allows users to do simple mathematic operations. 
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- React JS
 
-### `npm start`
+## Getting Started
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+**To get a local copy up and running follow these simple example steps.**
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Prerequisites
+A basic understanding of Git, Github, and React JS
 
-### `npm test`
+You also need an editor like VS Code or Sublime to edit project files
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Install
+1. On your computer, open your terminal (For Mac and Linux) or CMD (for Windows) 
+2. Navigate to the directory where you would like to run this project from, preferably from your web server directory
+3. Type the following command
 
-### `npm run build`
+> git clone https://github.com/rtonata88/calculator.git
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Usage
+Once the project folder is downloaded to your computer, user `npm run start` to run the project in the browser.
 
```

Comment:
- [ ] Technically, users cannot run your application (`npm run start`) without installing the packages ie `npm install`. Kindly add that as a necessary instruction for your application.